### ISSUE TITLE
Move CSS reset code to a reset.css file

### ISSUE THREAD

**Issue by cliftondavies** [cliftondavies/New-York-Times#7]
Store CSS reset independent of styles.css

**Comment by jydoskey** [comment]
Hello @cliftondavies, I have gone through your codebase and I will like to help work on this issue.

**Comment by cliftondavies** [comment]
That's brilliant @jydoskey! You can fork the repo and create a feature branch. Submit a pull request once you're done.

**Comment by jydoskey** [comment]
Just submitted a pull request for your review and approval. Thanks

**Comment by cliftondavies** [comment]
Hey man. I just merged your pull request. Awesome work!  💯 🙌 
Only minor thing is that the reset link should have come before the main styles.css, in the index.html file, for the page to render properly. I've swapped them now.  ✔️ 
I'll connect with you on Github to be in touch.
Keep working toward making that vision a reality. Maybe we can collaborate somewhere down the line.   😃 🤝 
Thanks again!  🙏
